### PR TITLE
log saves language info

### PR DESCRIPTION
**Pull request by jorgegorka**
Now logs save the current text like "patient added" but if we change language we want to see the logs in our language.

This PR saves the I18n key and will translate it when displaying the message in the active language.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"Metrics/LineLength":
+  Max: 140
```
Comment (tripledub) [inline .rubocop.yml]: Very generous ;)

File: cdx_core/app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# Application controller. Where it all starts.
```
Comment (tripledub) [inline cdx_core/app/controllers/application_controller.rb]: Is this just to keep `RuboCop` happy?

Comment (jorgegorka) [inline cdx_core/app/controllers/application_controller.rb]: It is. Although it is a good practice.

File: cdx_core/app/controllers/patients_controller.rb [hunk 3]
```diff
@@ -38,18 +40,18 @@ def new
...
     @patient.site = @navigation_context.site
 
-    if validate_name_and_entity_id && @patient.save_and_audit(current_user, I18n.t('patients.create.audit_log', patient_name: @patient.name))
+    if validate_name_and_entity_id && @patient.save_and_audit(current_user, "t{patients.create.audit_log}: #{@patient.name}")
       next_url = if params[:next_url].blank?
-        patient_path(@patient)
-      else
-        "#{params[:next_url]}#{params[:next_url].include?('?') ? '&' : '?'}patient_id=#{@patient.id}"
-      end
+                   patient_path(@patient)
```
Comment (tripledub) [inline cdx_core/app/controllers/patients_controller.rb]: This looks a bit odd

Comment (jorgegorka) [inline cdx_core/app/controllers/patients_controller.rb]: It is formated under if.  It looks good on the file, not so good in a git diff.

File: cdx_core/spec/controllers/comments_controller_spec.rb [hunk 4]
```diff
@@ -102,13 +104,13 @@
...
 
-        expect(response).to  render_template('show')
+        expect(response).to render_template('show')
       end
     end
   end
 
   context 'user with no edit patient permission' do
-    let(:invalid_user)           { Institution.make.user }
+    let(:invalid_user) { Institution.make.user }
```
Comment (tripledub) [inline cdx_core/spec/controllers/comments_controller_spec.rb]: +1 I never liked that style anyway :) :)

**Comment by tripledub** [comment]
Nothing to offend the `StewboCop` here, unless we are going to add `RuboCop` to the build though I wouldn't bother adding top comments that just state the name of the class. I have the `RuboCop` linter in my `vim` but just use it as a personal guide and usually ignore errors like that

File: cdx_core/app/controllers/application_controller.rb (hunk 2 above)
Comment (jackregnart) [inline cdx_core/app/controllers/application_controller.rb]: Funny, I also played with `RuboCop` today. That's a check I turned off ;)

Comment (tripledub) [inline cdx_core/app/controllers/application_controller.rb]: I can see the value in service objects etc where an explanation may be useful but not in boilerplate idiomatic rails classes

Comment (jackregnart) [inline cdx_core/app/controllers/application_controller.rb]: Yeah you're right. I guess it's Ruby specific, that's why it's not `RailsieCop` :)

**Pull request closed without merging**